log/log.go

```diff
@@ -33,36 +28,35 @@ func Color(code, msg string) string {
 }
 
 const (
-	debugLog    = iota
+	traceLog uint8 = iota
+	debugLog
 	infoLog
```

Comment:
Need to modify config.json.sample and the other config.json files.

Reply:
traceLog has been removed in the next commit, so the log levels in config.json can stay unchanged.